### PR TITLE
Added new env variable and speech function for alternative TTS voice

### PR DESCRIPTION
**Pull request by meta-fx**
### Background
This change was introduced to provide users with an alternative TTS voice option using the StreamElements API. It allows users to switch between the default TTS voice and the new alternative voice easily.

### Changes
1.) Added a new brian_speech function to the speech.py file, which utilizes the StreamElements API for an alternative TTS voice.
2.) Added a new environment variable and modified the config.py file to account for the new environment variable so that users to choose between the default TTS voice and the alternative voice.

### Documentation
The changes made in this pull request are documented in the code through comments, explaining the purpose of the new function and the environment variable.

### Test Plan
1.) Tested the new function with various text inputs to ensure that the alternative TTS voice works as expected.
2.) Tested the environment variable by toggling its value and verified that the selected TTS voice is used accordingly.

### PR Quality Checklist
- [✅] My pull request is atomic and focuses on a single change.
- [✅] I have thouroughly tested my changes with multiple different prompts.
- [✅] I have considered potential risks and mitigations for my changes.
- [✅] I have documented my changes clearly and comprehensively.
- [✅] I have not snuck in any "extra" small tweaks changes

### CODE REVIEW

**Comment by alexonab** [comment]
There seem to be unnecessary new lines added that are not needed for the proposed change functionality.

**Comment by meta-fx** [comment]
> There seem to be unnecessary new lines added that are not needed for the proposed change functionality.

Yeah I noticed that afterwards, my editor formats my code and adds unnecessary lines. I need to fix that and try again.

File: scripts/config.py [hunk 1]
```diff
@@ -69,7 +72,7 @@ def __init__(self):
 
         # User agent headers to use when browsing web
         # Some websites might just completely deny request with an error code if no user agent was found.
-        self.user_agent_header = {"User-Agent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.97 Safari/537.36"}
+        self.user_agent_header = {"User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.97 Safari/537.36"}
```
Comment (nponeccop) [inline scripts/config.py]: This change is unrelated. Roll it back.

File: scripts/config.py [hunk 2]
```diff
@@ -131,4 +134,4 @@ def set_pinecone_region(self, value: str):
 
     def set_debug_mode(self, value: bool):
         """Set the debug mode value."""
-        self.debug_mode = value
+        self.debug_mode = value
```
Comment (nponeccop) [inline scripts/config.py]: This is an unrelated change. Rollback (or add back the end-of-file line terminator that you or your editor deleted).

File: scripts/speak.py [hunk 3]
```diff
@@ -50,11 +67,15 @@ def macos_tts_speech(text):
     os.system(f'say "{text}"')
 
 def say_text(text, voice_index=0):
-
+
```
Comment (nponeccop) [inline scripts/speak.py]: You introduced trailing space. Remove it. (It sometimes creeps in during merges)

File: scripts/speak.py [hunk 4]
```diff
@@ -66,4 +87,4 @@ def speak():
 
     queue_semaphore.acquire(True)
     thread = threading.Thread(target=speak)
-    thread.start()
+    thread.start()
```
Comment (nponeccop) [inline scripts/speak.py]: Your editor keep removing the CRLF at the end of file. Configure it not to do it, e.g. with .editorconfig

**Review by nponeccop — APPROVED** [review]

File: .env.template [hunk 5]
```diff
@@ -13,3 +13,4 @@ OPENAI_AZURE_DEPLOYMENT_ID=deployment-id-for-azure
 IMAGE_PROVIDER=dalle
 HUGGINGFACE_API_TOKEN=
 USE_MAC_OS_TTS=False
+USE_BRIAN_TTS=False
```
Comment (nponeccop) [inline .env.template]: Add a final newline

**Review by nponeccop — APPROVED** [review]

**Comment by nponeccop** [comment]
@meta-fx There are conflicts now

**Review by nponeccop — CHANGES_REQUESTED** [review]
You still need to add that newline you removed back. Just press Enter in your editor at the endo of .env.template once. People keep adding and removing newlines creating conflicts. It isn't good, and the right way is to have the final line separator. Some editors keep removing it, vim insist in adding it.

File: .env.template [hunk 6]
```diff
@@ -102,10 +102,6 @@ CUSTOM_SEARCH_ENGINE_ID=your-custom-search-engine-id
 # USE_MAC_OS_TTS - Use Mac OS TTS or not (Default: False)
 USE_MAC_OS_TTS=False
 
-### ELEVENLABS
```
Comment (richbeales) [inline .env.template:105]: Is the removal of these ELEVENLABS elements intentional?

Comment (meta-fx) [reply .env.template:105]: Sorry it was not intentional.

**Review by nponeccop — DISMISSED** [review]

**Review by richbeales — APPROVED** [review]

**Comment by richbeales** [comment]
please fix flake8 

scripts/speak.py:55:1: E302 expected 2 blank lines, found 1
scripts/speak.py:72:1: E302 expected 2 blank lines, found 1

then this can be merged.  Thanks

**Review by nponeccop — APPROVED** [review]

**Review by p-i- — APPROVED** [review]
